src/controllers/auth.js

```diff
@@ -21,8 +21,10 @@ export default {
    */
   signup: async (request, response) => {
     const existingUser = await Users.getExistingUser(request.body.email);
```

Comment:
`Nit`: It would be nicer to destructure request.body i.e

```javascript
const { email, userName } = request.body
```

instead of always doing `request.body.email` etc